Arduino/McLighting/request_handlers.h

```diff
@@ -1,6 +1,37 @@
 // ***************************************************************************
 // Request handlers
 // ***************************************************************************
+#ifdef ENABLE_E131
+void handleE131(){
+  if (!e131.isEmpty())
+  {
+    e131_packet_t packet;
+    e131.pull(&packet); // Pull packet from ring buffer
+
+    uint16_t universe = htons(packet.universe);
+    uint8_t *data = packet.property_values + 1;
+
+    if (universe < UNIVERSE || universe > UNIVERSE_COUNT) return; //async will take care about filling the buffer
```

Comment:
This is wrong condition:

```
    if (universe < UNIVERSE || universe >= UNIVERSE+UNIVERSE_COUNT) return; //async will take care about filling the buffer
```

Reply:
This condition is there to take care of data coming higher than UNIVERSE_COUNT.

Reply:
Then the comment is misleading: `Total number of Universes to listen for, starting at UNIVERSE`. I would rather name it `START_UNIVERSE` and `LAST_UNIVERSE/END_UNIVERSE`. `COUNT` indiciates that something is relative, not absolute.

Reply:
Got it, I'll make those changes.

Reply:
https://github.com/toblum/McLighting/commit/2551240e634a77b1896ba63c0bfa79b1c543ee55#diff-cc8f4c02683cfee4157e0193bb932853R16